### DISCUSSION TITLE
Do I have to push the YML to master branch for the action to be executed?

### DISCUSSION THREAD

**Discussion by matias-quezada** [community/community#25317] [Actions]
<p>I’ve a private project where <code>master</code> is deploying to production through a different CI system. So I can’t push to master without running a deploy to production which is not an option.</p>
<p>I’ve pushed this workflow to <code>development-kubernetes</code> branch but nothing is appearing in the Actions tab.</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - master
      - release
      - development
      - development-kubernetes
    paths:
      - 'build/CENSORED-client*'
      - 'CENSORED.Client/*'

name: CENSORED-client

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Azure Login
        uses: azure/login@v1
        with:
          creds: ${{ secrets.AZURE_CREDENTIALS }}

      - name: Azure CLI script
        uses: azure/CLI@v1
        with:
          azcliversion: 2.0.72
          inlineScript: $GITHUB_WORKSPACE/deploy-script.sh CENSORED-client $GITHUB_SHA
</code></pre>
<p>I pushed the same file to the <code>master</code> branch of a different repository and I can see the new workflow in the actions tab.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Did that push also change any of the files matched by your <code>paths</code> list? Because if not that’s your reason right there. :wink:</p>

**Comment by matias-quezada** [comment]
<p>I added the workflow YML itself to the list of files matched in the path list and worked. Thank you! :smiley:</p>
